src/Microsoft.VisualStudio.Editors/PropPages/xlf/PackagePropPage.cs.xlf

```diff
@@ -137,6 +132,121 @@
         <target state="translated">V&amp;erze souboru sestavení:</target>
         <note />
       </trans-unit>
+      <trans-unit id="PackageLicenseLabel.Text">
+        <source>&amp;Licensing:</source>
+        <target state="new">&amp;Licensing:</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="ExpressionLabel.Text">
+        <source>&amp;Expression:</source>
+        <target state="new">&amp;Expression:</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="OrLabel.Text">
+        <source>&amp;OR</source>
+        <target state="new">&amp;OR</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="FileLabel.Text">
+        <source>&amp;File:</source>
+        <target state="new">&amp;File:</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="LicenseBrowseButton.Text">
+        <source>Browse...</source>
+        <target state="new">Browse...</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Copyright.AccessibleName">
+        <source>Package Copyright</source>
```

Comment:
The items from here down do not have `&`-style accelerators. Are we supposed to add them?

Reply:
I don't believe so - These are auto generated.  Just to make sure though: @tmeschter

Reply:
Don't worry about the accelerators in the .xlf files. However, have proper accelerators been defined for the newly added controls?

Reply:
Working on that - I am having trouble even getting the old accelerators responding correctly